packages/gatsby/src/schematics/application/application.spec.ts

```diff
@@ -282,18 +282,60 @@ describe('app', () => {
       appTree
     );
 
-    const eslintJson = readJsonInTree(tree, '/apps/my-app/.eslintrc.json');
     const packageJson = readJsonInTree(tree, '/package.json');
-
-    expect(eslintJson.extends).toEqual(
-      expect.arrayContaining(['plugin:@nrwl/nx/react'])
-    );
     expect(packageJson).toMatchObject({
       devDependencies: {
         'eslint-plugin-react': expect.anything(),
         'eslint-plugin-react-hooks': expect.anything(),
       },
     });
+
+    const eslintJson = readJsonInTree(tree, '/apps/my-app/.eslintrc.json');
+    expect(eslintJson).toMatchInlineSnapshot(`
```

Comment:
This diff is purely the change to make the snapshot to capture the whole config, instead of just `extends`